Classes/Neos/Neos/Ui/Domain/Model/Feedback/Operations/RemoveNode.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace Neos\Neos\Ui\Domain\Model\Feedback\Operations;
+
+use Neos\Neos\Ui\Domain\Model\FeedbackInterface;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\Flow\Mvc\Controller\ControllerContext;
+
+class RemoveNode implements FeedbackInterface
```

Comment:
Agreeing to Wilhelms thoughts here :) Instead of "AddDocumentNode", I'd rather use more UI-specific namings like "LoadDocumentNodeToStore" or so...